bigframes/session/__init__.py

```diff
@@ -1455,10 +1461,18 @@ def _create_empty_temp_table(
     def _create_total_ordering(
         self,
         table: ibis_types.Table,
+        table_size: int,
```

Comment:
Likewise, we might need to handle missing size here.

Reply:
added.